### ISSUE TITLE
Site to be unblocked

### ISSUE THREAD

**Issue by TechComputerServiceDunkerque** [ragibkl/adblock-dns-server#54]
here is already a site to unlock, leave the ticket open :)

www.sfr.fr 
www.leclercdrive.fr

**Comment by ragibkl** [comment]
I have added both sites in my whitelist.

**Comment by TechComputerServiceDunkerque** [comment]
Is leclercdrive.fr still blocked? I can't access it.

**Comment by ragibkl** [comment]
I have already whitelisted `leclercdrive.fr`. It returns the same answer as Google DNS now.
I don't know, maybe the site is down?

I've also whitelisted `coronastatus.fr` now.

**Comment by ragibkl** [comment]
It seems that `coronastatus.fr` don't have an active dns record right now. I've whitelisted it anyway, but maybe it's not a working site.

**Comment by TechComputerServiceDunkerque** [comment]
I think the corona site is in creation, a French newspaper published it too quickly. After for leclercdrive, with Cloudflare's dns 1.1.1, it works well but with Bancuh no, weird. Otherwise I come back in test, I'm really satisfied with the speed of your dns :), I have no more bugs :). Except a few sites that block, but I'm preparing a list :)

I think you should remove the DNS from google, a lot of people complain because they block some websites. Why don't you put the one from cloudflare, it's much faster and maybe your server would go faster?

**Comment by ragibkl** [comment]
Hi @TechComputerServiceDunkerque ,

Sorry for late reply. I've setup the dns server in france to use `1.1.1.1` for now. Let's see how well it performs.

Apparently, I also need to whitelist `accueil.leclercdrive.fr`. I have done this now. let's see if it works.

**Comment by TechComputerServiceDunkerque** [comment]
It works very well, eleclerc works now.  I'm preparing a list for you to unlock.  Thanks for the work.

**Comment by TechComputerServiceDunkerque** [comment]
unblocking site: 

cdburnerxp.se
Msn.com
listen.deezer.com
football365.fr
c.eu15.content.force.com

**Comment by ragibkl** [comment]
Hi @TechComputerServiceDunkerque 

The following had to be whitelisted:
- cdburnerxp.se, www.cdburnerxp.se
- a-0003.a-msedge.net (CNAME of www.msn.com)
- football365.fr, www.football365.fr
- c.eu15.content.force.com
- rts.ch, www.rts.ch

The following is not in my blocklist. I think it should just work without changes:
- forums.commentcamarche.net

The following does not look like valid domains. Even if you use `1.1.1.1` dns, they do not work:
- listen.deezer.com (www.deezer.com is working)

I think that should about cover it. Let me know if it is still not working.

**Comment by TechComputerServiceDunkerque** [comment]
Hello ragibkl, 

how are you? Everything is fine with your French server, I put it to friends and family and everybody says it's good because it blocks 99.9% of the ads, but there are still ads that can annoy the world, would you manage to block the sources of the ads on this website ? 

https://ed-protect.org/v8z0nPD

thank you

**Comment by ragibkl** [comment]
@TechComputerServiceDunkerque ,
I am glad that you and your friends and family are finding the dns server in France to be useful. Also, thank you for all the efforts that you are putting as well in reporting issues and making suggestions. Parts of the improvements on this project are from your efforts as well. Well Done.

With regard to `https://ed-protect.org/v8z0nPD` link, what do you need to do here? I don't see any ads appear when I open the page.

**Comment by TechComputerServiceDunkerque** [comment]
Strange, a few days ago, I had a lot of ads on this site. Maybe the people in charge of the site removed them. I will continue to help you with the 😊 server.

**Comment by TechComputerServiceDunkerque** [comment]
Hello, it's possible to unblock these sites, thank you :)

c.eu15.content.force.com
click.sfm.bebeboutik.com
awin1.com
awstrack.me
download.cdn.yandex.net
filehippo.com

possible to unblock this spam site

knockandget1.live

Merci

**Comment by ragibkl** [comment]
I've unblocked the following:
- c.eu15.content.force.com
- click.sfm.bebeboutik.com
- awin1.com
- download.cdn.yandex.net
- filehippo.com

I think, the following does not exist:
- knockandget1.live
- awstrack.me

**Comment by TechComputerServiceDunkerque** [comment]
Hello, here is a new site to unlock.  

https://www.winamax.fr/

Thank you

**Comment by ragibkl** [comment]
> Hello, here is a new site to unlock.
> 
> https://www.winamax.fr/
> 
> Thank you

Hi @TechComputerServiceDunkerque , sorry for late reply, but I made the code changes a few hours after you posted this. So it should work now. I only forgot to update you about it.

**Comment by TechComputerServiceDunkerque** [comment]
Merci, ça refonctionne

**Comment by TechComputerServiceDunkerque** [comment]
Hello, here is a new site to unlock.

https://www.awin1.com/

Merci

**Comment by ragibkl** [comment]
> Hello, here is a new site to unlock.
> 
> https://www.awin1.com/
> 
> Merci

Should be done now.

**Comment by TechComputerServiceDunkerque** [comment]
https://www.iobit.com
http://update.iobit.com
http://majax31isnotdown.blogspot.com

Merci

**Comment by ragibkl** [comment]
> https://www.iobit.com
> http://update.iobit.com
> http://majax31isnotdown.blogspot.com
> 
> But for awin1, it still doesn't work.
> 
> Merci

Just done this. Let me know if it's not working.

**Comment by ragibkl** [comment]
> https://appgallery.cloud.huawei.com/appdl/C27162

I made the changes just now. Let me know if not working.

**Comment by ragibkl** [comment]
> https://appgallery.cloud.huawei.com/appdl/C27162

I made the changes now. Let me know if not working.

**Comment by TechComputerServiceDunkerque** [comment]
Bonjour Ragibkl,

how are you doing? Is it possible to remove all the hosts and just put these lists there? 

Because I see a lot of people complaining that the hosts I gave you are blocking a lot of websites when they shouldn't. Unless you know which host is blocking websites. Thanks to you for the work you do. 

https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
https://mirror1.malwaredomains.com/files/justdomains
https://s3.amazonaws.com/lists.disconnect.me/simple_malvertising.txt
http://sysctl.org/cameleon/hosts
https://s3.amazonaws.com/lists.disconnect.me/simple_tracking.txt
https://s3.amazonaws.com/lists.disconnect.me/simple_ad.txt
https://justdomains.github.io/blocklists/lists/easylist-justdomains.txt
https://justdomains.github.io/blocklists/lists/easyprivacy-justdomains.txt
https://justdomains.github.io/blocklists/lists/adguarddns-justdomains.txt
https://justdomains.github.io/blocklists/lists/nocoin-justdomains.txt
https://adaway.org/hosts.txt
https://gitlab.com/quidsup/notrack-blocklists/raw/master/notrack-blocklist.txt
https://raw.githubusercontent.com/crazy-max/WindowsSpyBlocker/master/data/hosts/spy.txt

**Comment by ragibkl** [comment]
Hi @TechComputerServiceDunkerque ,

I can do that no problem. I'll make the changes now. You can just report back to me with any suggestions to change the hosts.

**Comment by TechComputerServiceDunkerque** [comment]
Thank you, it's really nice

**Comment by TechComputerServiceDunkerque** [comment]
Hello, 

i can't update the virus base of Avira antivir and KungPeng from Total 360 security with the dns. Is it possible to unblock?  Thank you

**Comment by ragibkl** [comment]
Hi @TechComputerServiceDunkerque , sorry for late reply. I got busy these few days.

> Hello,
> 
> i can't update the virus base of Avira antivir and KungPeng from Total 360 security with the dns. Is it possible to unblock? Thank you

I'm not sure what are the domains used for Avira or KungPeng. I searched online if other users have this experience with other dns adblock projects, but to no avail.

If you know the domains to unblock, let me know.

Alternatively, are you able to run a copy of the adblock server on your own computers? If you can do that, you can read the server logs and figure out which domains were being blocked.

**Comment by Tomatoide** [comment]
https://www.omgubuntu.co.uk/ is broken, seems to be due to blocked cdn?
(pressablecdn.com)

**Comment by ragibkl** [comment]
Hi @Tomatoide , thanks for the report.

I've added the fix here: https://github.com/ragibkl/adblock-dns-server/commit/39b9ffb4b21ebf146209f498279ff0f3fc138989

**Comment by Tomatoide** [comment]
nytimes.com is broken, but couldn't find out the cause, also, energized lists in blacklist src json are 404

**Comment by ragibkl** [comment]
Hi @Tomatoide ,

Currently, we have 2 ways to debug issues in broken sites.

On web browser, you can do right click, and inspect element. Then go to networks tab. You might need to reload the page after that. You can usually find list of assets where the network request failed. This is due to the dns adblock giving false info.

Another way, is to visit the logs viewer page. Example of the following server:

server: dns1.bancuh.com

dns
ipv4: 139.59.219.245
ipv6: 2400:6180:0:d0::f7:1001

logs-viewer
ipv4: http://139.59.219.245:8080/
ipv6: http://[2400:6180:0:d0::f7:1001]:8080/

The logs viewer should show you a list of domains that have been blocked. It might tell us some clues as to what domains need to be whitelisted.

**Comment by ragibkl** [comment]
I should probably update the readme file with the debug steps above. I've added some changes to fix for nytimes. do check on your side @Tomatoide .

**Comment by ragibkl** [comment]
Hi guys.
This issue thread is getting too long and unwieldy to manage. I will close this issue for now.
Please open a new issue if you find any legitimate sites to whitelist.
